process.py

```diff
@@ -386,9 +387,15 @@ def get_provider_code(directory_name=None):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--path', type=str, default="static/data/lms")
```

Comment:
I actually think it's a good idea to keep the `--path` argument so we can always load everything from a directory that is not necessarily `static/data/lms`.